assets/js/googlesitekit/datastore/site/reset.test.js

```diff
@@ -33,32 +33,26 @@ import {
 import { STORE_NAME } from './constants';
 
 describe( 'core/site reset', () => {
-	let apiFetchSpy;
 	let registry;
 
 	beforeEach( () => {
 		registry = createTestRegistry();
-
-		apiFetchSpy = jest.spyOn( { apiFetch }, 'apiFetch' );
 	} );
 
 	afterEach( () => {
 		unsubscribeFromAll( registry );
-		apiFetchSpy.mockRestore();
+		fetchMock.restore();
+		fetchMock.mockClear();
 	} );
 
 	describe( 'actions', () => {
 		describe( 'fetchReset', () => {
 			it( 'sets isDoingReset ', async () => {
 				const response = true;
-				fetch
-					.doMockOnceIf(
-						/^\/google-site-kit\/v1\/core\/site\/data\/reset/
-					)
-					.mockResponseOnce(
-						JSON.stringify( response ),
-						{ status: 200 }
-					);
+				fetchMock.once(
+					/^\/google-site-kit\/v1\/core\/site\/data\/reset/,
+					{ body: JSON.stringify( response ), status: 200 }
```

Comment:
@aaemnnosttv It's not necessary for most responses. But it seems to be necessary for booleans, as it will error with:

` Google Site Kit API Error { code: 'invalid_json',
      message: 'The response is not a valid JSON response.' }`

If the response is just a boolean.

Alternatively we could change `const response = true` to `const response = 'true'`